### PR TITLE
Project to function; function tidyup; DGGS funcs CC

### LINKED ISSUE #192: Create an isolationed Conformance Class for DGGS support
See Issue #120 for possible duplication

### LINKED ISSUE #231: Function to project to a lower dimension
We need a toproject function to project to lower dimensions

### PR DESCRIPTION
**Pull request by nicholascar**
Closes #231
Closes #192

Also:

* fills in missing Simple Features Access functions
* tidies the Functions Summary Table 
* adds an Geo->SFA mapping table

### CODE REVIEW

**Comment by nicholascar** [comment]
@rggibb: you might like to review this one too

**Comment by situx** [comment]
The projectTo function seems good to me, but I have some issues with the GeoSPARQL to SFA table. There are functions which are in GeoSPARQL (e.g. asWKT) which are differently named in SFA. So we will likely not redefine these functions in GeoSPARQL.
geof:asWKT is equivalent to sfa asText for example
I think we should still mark this function as implemented in the comparison table

File: 1.1/spec/17-Annex-B.adoc [hunk 1]
```diff
@@ -6,76 +6,164 @@ This annex summarises all the functions defined in GeoSPARQL, providing descript
...
+
+== B.2 GeoSPARQL to SFA Functions Mapping
+
+The following table indicates which GeoSPARQL non-topological query functions map to Simple Features Access (<<ISO19125-1>>) functions and in which GeoSPARQL version the functions are defined.
+
+Where the Simple Features Access function has the same name as the GeoSPARQL function, 'x' is recorded.
+
+[role="smalltext"]
+|===
+| GeoSPARQL Function | in 1.0 | in 1.1 | SFA
```
Comment (mperry455) [inline 1.1/spec/17-Annex-B.adoc:135]: It looks like some of the labels may be wrong here. For instance, isn't maxX a 1.1 function?

Comment (nicholascar) [reply 1.1/spec/17-Annex-B.adoc:135]: I _think_ I've fixed the table!

**Review by mperry455 — APPROVED** [review]
Looks good.

**Comment by nicholascar** [comment]
> I think we should still mark this function as implemented in the comparison table
Done!

**Review by situx — APPROVED** [review]

**Pull request merged**